{"name": "Ручка /api/v1/courier", "status": "passed", "description": "Создаём курьера, затем создаём ещё одного курьера, с такими же данными. Должна быть неудача.", "start": 1710444109629, "stop": 1710444109748, "uuid": "70be64ba-8961-4458-89a8-cb4528ec55ac", "historyId": "2c26b9034b3d161dd45c110f60ac5882", "testCaseId": "2c26b9034b3d161dd45c110f60ac5882", "fullName": "test_api.tests.test_create_courier.TestCreateCourier#test_create_simular_couriers_error_409", "labels": [{"name": "parentSuite", "value": "test_api.tests"}, {"name": "suite", "value": "test_create_courier"}, {"name": "subSuite", "value": "TestCreateCourier"}, {"name": "host", "value": "LAPTOP-KS9V29SF"}, {"name": "thread", "value": "23240-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_api.tests.test_create_courier"}]}